src/tools/moveedge.cpp

```diff
@@ -12,25 +12,31 @@ MoveEdgeTool::MoveEdgeTool()
 
 void MoveEdgeTool::activated(EditorView* editor)
 {
+  // Hide grips when the edge tool is selected
   editor->setVerticesVisible(false);
+  // Start highlighting edges when tool is activated
```

Comment:
Hmm, I wonder if maybe we should move some of the general stuff into the caller -- we want hoverEnter and hoverLeave no matter which tool we're switching into or out of, and we can always set vertices and edges visible on deactivate.